### ISSUE TITLE
man page builder: definition list items are bold

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1498]
The definition list items are bolded in the output from the man page builder.  This is a problem when documenting command options, which should look like:

```
**-f**, **--foo**\=\ *argument*
    Option description.
```

where "-f" and "--foo" are bold, "argument" is italicized or underlined, and the "," and "=" should be book face.  However, for the current man page builder, all of "-f, --foo=" is bolded, which looks ugly.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1498
- Originally reported by: [arianaut](https://bitbucket.org/arianaut)
- Originally created at: 2014-06-27T01:13:16.499

**Comment by tk0miya** [comment]
In my short investigation:

It seems this is a bug of docutils.
rst2man.py generates the same result:

Input:

```
Test
====

**-f**, **--foo**\=\ *argument*
    Option description.
```

Output:

```
.\" Man page generated from reStructuredText.
.
.TH TEST  "" "" ""
.SH NAME
Test \-
.
.nr rst2man-indent-level 0
.
.de1 rstReportMargin
\\$1 \\n[an-margin]
level \\n[rst2man-indent-level]
level margin: \\n[rst2man-indent\\n[rst2man-indent-level]]
-
\\n[rst2man-indent0]
\\n[rst2man-indent1]
\\n[rst2man-indent2]
..
.de1 INDENT
.\" .rstReportMargin pre:
. RS \\$1
. nr rst2man-indent\\n[rst2man-indent-level] \\n[an-margin]
. nr rst2man-indent-level +1
.\" .rstReportMargin post:
..
.de UNINDENT
. RE
.\" indent \\n[an-margin]
.\" old: \\n[rst2man-indent\\n[rst2man-indent-level]]
.nr rst2man-indent-level -1
.\" new: \\n[rst2man-indent\\n[rst2man-indent-level]]
.in \\n[rst2man-indent\\n[rst2man-indent-level]]u
..
.INDENT 0.0
.TP
.B \fB\-f\fP, \fB\-\-foo\fP=\fIargument\fP
Option description.
.UNINDENT
.\" Generated by docutils manpage writer.
.
```

docutils generates term of definition list as bold string (`.B \fB\-f\fP, \fB\-\-foo\fP=\fIargument\fP`).

**Comment by tk0miya** [comment]
Fixed at 549b6e2.
Thank you for reporting
